src-ts/header/utility-selectors/UtilitySelector/ProfileSelector/profile-logged-in/ProfileLoggedIn.tsx

```diff
@@ -62,7 +54,6 @@ const ProfileLoggedIn: FC<ProfileLoggedInProps> = (props: ProfileLoggedInProps)
                             <IconOutline.XIcon />
                         </div>
                         <ProfilePanel
-                            refObject={ref}
```

Comment:
The ProfilePanel.tsx still requires this prop. We need to either remove it from the props or assign it.

Also, the profile panel isn't closing when you click the Account settings link. Perhaps this has something to do w/that?